packages/@aws-cdk/aws-rds/lib/cluster-engine.ts

```diff
@@ -224,11 +246,89 @@ class AuroraClusterEngine extends MySqlClusterEngineBase {
   }
 }
 
+/**
+ * The versions for the Aurora MySQL cluster engine
+ * (those returned by {@link DatabaseClusterEngine.auroraMysql}).
+ */
+export class AuroraMysqlEngineVersion {
+  /** Version "5.7.12". */
+  public static readonly VER_5_7_12 = AuroraMysqlEngineVersion.builtIn_5_7('12', false);
+  /** Version "5.7.mysql_aurora.2.03.2". */
+  public static readonly VER_2_03_2 = AuroraMysqlEngineVersion.builtIn_5_7('2.03.2');
+  /** Version "5.7.mysql_aurora.2.03.3". */
+  public static readonly VER_2_03_3 = AuroraMysqlEngineVersion.builtIn_5_7('2.03.3');
+  /** Version "5.7.mysql_aurora.2.03.4". */
+  public static readonly VER_2_03_4 = AuroraMysqlEngineVersion.builtIn_5_7('2.03.4');
+  /** Version "5.7.mysql_aurora.2.04.0". */
+  public static readonly VER_2_04_0 = AuroraMysqlEngineVersion.builtIn_5_7('2.04.0');
+  /** Version "5.7.mysql_aurora.2.04.1". */
+  public static readonly VER_2_04_1 = AuroraMysqlEngineVersion.builtIn_5_7('2.04.1');
+  /** Version "5.7.mysql_aurora.2.04.2". */
+  public static readonly VER_2_04_2 = AuroraMysqlEngineVersion.builtIn_5_7('2.04.2');
+  /** Version "5.7.mysql_aurora.2.04.3". */
+  public static readonly VER_2_04_3 = AuroraMysqlEngineVersion.builtIn_5_7('2.04.3');
+  /** Version "5.7.mysql_aurora.2.04.4". */
+  public static readonly VER_2_04_4 = AuroraMysqlEngineVersion.builtIn_5_7('2.04.4');
+  /** Version "5.7.mysql_aurora.2.04.5". */
+  public static readonly VER_2_04_5 = AuroraMysqlEngineVersion.builtIn_5_7('2.04.5');
+  /** Version "5.7.mysql_aurora.2.04.6". */
+  public static readonly VER_2_04_6 = AuroraMysqlEngineVersion.builtIn_5_7('2.04.6');
+  /** Version "5.7.mysql_aurora.2.04.7". */
+  public static readonly VER_2_04_7 = AuroraMysqlEngineVersion.builtIn_5_7('2.04.7');
+  /** Version "5.7.mysql_aurora.2.04.8". */
+  public static readonly VER_2_04_8 = AuroraMysqlEngineVersion.builtIn_5_7('2.04.8');
+  /** Version "5.7.mysql_aurora.2.05.0". */
+  public static readonly VER_2_05_0 = AuroraMysqlEngineVersion.builtIn_5_7('2.05.0');
+  /** Version "5.7.mysql_aurora.2.06.0". */
+  public static readonly VER_2_06_0 = AuroraMysqlEngineVersion.builtIn_5_7('2.06.0');
+  /** Version "5.7.mysql_aurora.2.07.0". */
+  public static readonly VER_2_07_0 = AuroraMysqlEngineVersion.builtIn_5_7('2.07.0');
+  /** Version "5.7.mysql_aurora.2.07.1". */
+  public static readonly VER_2_07_1 = AuroraMysqlEngineVersion.builtIn_5_7('2.07.1');
+  /** Version "5.7.mysql_aurora.2.07.2". */
+  public static readonly VER_2_07_2 = AuroraMysqlEngineVersion.builtIn_5_7('2.07.2');
+  /** Version "5.7.mysql_aurora.2.08.0". */
+  public static readonly VER_2_08_0 = AuroraMysqlEngineVersion.builtIn_5_7('2.08.0');
```

Comment:
2.08.1 was released two weeks back.

Reply:
Really? I got these from running `aws rds describe-db-engine-versions --engine aurora-mysql --query "DBEngineVersions[].EngineVersion"`, and I don't see `2.08.1` there... I would love to know where you get your info from 🙂

Reply:
Hmm.. Not sure. I got my info from: https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/AuroraMySQL.Updates.2081.html

Reply:
Alright, will add!